### ISSUE TITLE
interact hook should take collections/topics/qrels flags

### ISSUE THREAD

**Issue by cmacdonald** [osirrc/jig#114]
If I want to repeat a run in my interact hook, its a bit difficult as the topics/qrels arent mounted. Similarly with which collections are in use.

**Comment by arjenpdevries** [comment]
I agree it would be good if `interact` mode would mount topics and qrels directories.

Maybe we need an environment variable for the information about collections that were created? (E.g., store the JSON version of the parameters?)

**Comment by Chriskamphuis** [comment]
It could be useful if they are already mounted during the `prepare` step, then they are available for both `search` and `interact`. Saving the image after the `search` step would also solve this issue, but then the `interact` can only use the topics/qrels after `search`.

**Comment by cmacdonald** [comment]
@Chriskamphuis I think the idea of the images by @r-clancy is that they can handle multiple collections in the same container image. hence why they are not mounted once in prepare?

**Comment by Chriskamphuis** [comment]
@cmacdonald Sure, we also support indexing multiple collection in one collection. But in a similar fashion it also could be possible to support multiple topic/qrels files right?
